README.md

```diff
@@ -1,38 +1,26 @@
 # Mews fiscalizations
 
-This is a repo with all the fiscalizations.
-It is a work in progress at this moment. You will find the individual fiscalizations in their folder and each of them has their own readme file and nuget package.
+This repository contains multiple projects. Each project supports reporting of e-invoices to the corresponding country's government authority.
 
-# Build and tests
+**Please note that we're not responsible for how these libraries are used and if the data provided is correct or not.**
+**Also, it is important to note that these libraries were created to meet our usages, so they might be missing some data that can be important in your case (feel free to extend any library and create a PR).**
+
+P.S Each project folder has it's own documentation (as described below in section **Projects**).
```

Comment:
Let's also put this sentence after the first line: `...ntry's government authority.`

```suggestion
Each project folder has it's own documentation. Use the link in the table below to get there.
```